### PR TITLE
WIP: My branch

### PR DESCRIPTION
**Pull request by laurentheirendt**

### CODE REVIEW

**Comment by laurentheirendt** [comment]
@Trefex, please review! 👍

**Comment by Trefex** [comment]
Full of mistake !!

File: _attendees/michaelJackson.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+name: Michael Jackson
+position: Position
+group: GroupName
+institution: University of Luxembourg
+ghurl: https://github.com/<yourUserName>
+ghimg: https://avatars2.githubusercontent.com/u/44060023
```
Comment (Trefex) [inline _attendees/michaelJackson.md]: Not sure I understand this.

**Comment by Julien-uni** [comment]
FYI: MJ is dead :(

**Pull request merged**